custom_components/places/tracker.py

```
@@ -23,7 +23,16 @@
 
 
 def _float_or_none(value: Any) -> float | None:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)